src/ape/contracts/base.py

```diff
@@ -468,6 +468,46 @@ def __len__(self):
         logs = self.provider.get_contract_logs(self.log_filter)
         return sum(1 for _ in logs)
 
+    def __call__(self, *args: Any, **kwargs: Any) -> MockContractLog:
+        # Create a dictionary from the positional arguments
+        event_args = dict(zip((input.name for input in self.abi.inputs), args))
+
+        overlapping_keys = set(k for k in event_args.keys() if k is not None) & set(
+            k for k in kwargs.keys() if k is not None
+        )
+
+        if overlapping_keys:
+            raise ValueError(
+                f"Overlapping keys found in arguments: '{', '.join(overlapping_keys)}'."
+            )
+
+        # Update event_args with keyword arguments
+        event_args.update(kwargs)  # type: ignore[arg-type]
```

Comment:
rather than a type ignore here, you can do `evert_args: Dict` at the top to tell mypy to not care about the types of the elements.

Reply:
I'm still getting the type error unless I use `event_args: Dict[Any, Any]`, just `Dict` still pops the same error